addons/controllers/cpi/vspherecpiconfig_controller.go

```diff
@@ -36,6 +37,7 @@ type VSphereCPIConfigReconciler struct {
 
 //+kubebuilder:rbac:groups=cpi.tanzu.vmware.com,resources=vspherecpiconfigs,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=cpi.tanzu.vmware.com,resources=vspherecpiconfigs/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=vmware.infrastructure.cluster.x-k8s.io,resources=providerserviceaccounts,verbs=get;create;list;watch;update;patch
```

Comment:
Could we check if existing rbac for [addons-manager](https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/management/addons-manager-v2/bundle/config/upstream/addons-manager.yaml) needs to be updated?

Reply:
yes, looks like it needs to be updated with these additional rules brought by the VSphereCPIConfigController. I will update it as well

Reply:
updated.